src/hooks/useSnackMessage.ts

```diff
@@ -17,65 +22,61 @@ interface SnackInputs {
     headerTxt?: string;
     headerId?: string;
     headerValues?: Record<string, string>;
+    persistent?: boolean;
```

Comment:
Why don't we just extend notistack type ? Then it would be persist, not persistent. But I think we shouldn't be intrusive into the notistack API, we just want to extend it a bit

Reply:
i exetended OptionsObject interface that define the attributes we want